data/liveinst/liveinst

```diff
@@ -33,6 +46,19 @@ if [ -z "$DBUS_SESSION_BUS_ADDRESS" ]; then
     export DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/${PKEXEC_UID}/bus
 fi
 
+# pkexec clears WAYLAND_DISPLAY from environment
+if [ -z "$WAYLAND_DISPLAY" ] && [ -e "$WAYLAND_DISPLAY_SOCKET" ]; then
+    WAYLAND_DISPLAY=$(cat "${WAYLAND_DISPLAY_SOCKET}")
+    export WAYLAND_DISPLAY
+fi
+
+# use the correct home and config directories for system settings
+PKEXEC_USER=$(id -un "${PKEXEC_UID}")
```

Comment:
On Workstation I'm getting:
```
id -un ''
id: '': no such user
```

The whole Web UI won't start at all.

Reply:
how is it that PKEXEC_USER isn't defined? It's a polkit variable...

Reply:
Re-tested on official Workstation image from today, the same issue happens there.

Reply:
For clarification my fix will remove the errors but most probably it won't fix the fact that this change won't allow Anaconda to start.

My PR is already merged so feel free to rebase and that should simplify debugging of this.

Reply:
Done.